flag_int64_slice.go

```diff
@@ -131,11 +137,56 @@ func (c *Context) GlobalInt64Slice(name string) []int64 {
 func lookupInt64Slice(name string, set *flag.FlagSet) []int64 {
 	f := set.Lookup(name)
 	if f != nil {
-		parsed, err := (f.Value.(*Int64Slice)).Value(), error(nil)
-		if err != nil {
-			return nil
+		if value, ok := f.Value.(*Int64Slice); ok {
```

Comment:
Nit/optional: Generally it's better to keep the error scenario in the conditional. We get to remove a level of indentation and reduce the overall cognitive overhead of the function:

```go
value, ok := f.Value.(*Int64Slice)
if !ok {
	return nil
}
// regular logic goes here ...
```

Reply:
Same optional note for the other two types.